apps/avatax/src/logger.ts

```diff
@@ -3,9 +3,7 @@ import packageJson from "../package.json";
 
 logger.settings.maskValuesOfKeys = ["metadata", "username", "password", "apiKey"];
 
-if (process.env.NODE_ENV !== "production") {
```

Comment:
we should probably have this `if`

`if(process.env.NODE_ENV !== "production" || process.env.ENABLE_CONSOLE_LOGS==="true")`